open-x-htb-system-tests.js

```diff
@@ -0,0 +1,192 @@
+'use strict';
+
+function getPartnerId() {
+    return 'OpenXHtb';
+}
+
+function getStatsId() {
+    return 'OPNX';
+}
+
+function getCallbackType() {
+    return 'NAME';
+}
+
+function getArchitecture() {
+    return 'SRA';
+}
+
+function getConfig() {
+    return {
+        1: {
+            adUnitId: '54321',
+            sizes: [[300, 250], [300, 600]]
+        },
+        2: {
+            adUnitId: '12345',
+            sizes: [[300, 600]]
+        },
+        3: {
+            adUnitId: '654321',
+            sizes: [[728, 90]]
+        }
+    };
+}
+
+function getBidRequestRegex() {
+    return {
+        method: 'GET',
+        urlRegex: /.*openx\.net\/w\/1\.0\/arj\?.*/
+    };
+}
+
+function validateBidRequest(request) {
+    // Check query string parameters.
+    expect(request.query.auid).toBe('54321,12345,654321');
+    expect(request.query.aus).toBe('300x250,300x600|300x600|728x90');
+}
+
+function getValidResponse(request, creative) {
```

Comment:
Must ensure that for both `getValidBidResponse()` and `getValidBidResponseWithDeal()` that the winning bid’s `pub_rev` property is set such that it converts to 200 cents.

Reply:
I am unable to reproduce the following behaviour.

>for getValidBidResponseWithDeal() pub_rev:1000 converts to ix_ox_pm:<size>_**1000**, seems not using the same scale.

The `pub_rev` value should not affect the `ix_ox_pm` key, it should only affect the `ix_ox_om` key.

As a test, I changed the `pub_rev` to `'1000'` and `'2000'` respectively in `getValidBidResponseWithDeal()` and then placed a breakpoint at the top of `validateTargetingWithDeal()`. Here is a screenshot of what I am seeing:

<img width="561" alt="screen shot 2018-10-18 at 4 27 35 pm" src="https://user-images.githubusercontent.com/28928533/47182195-dad34e80-d2f2-11e8-9cc5-14d819d33d8b.png">

Here the `ix_ox_om=300x250_100` is as I expected for a `pub_rev:'1000'`, and the `ix_ox_pm=300x250_1` for a `dealid:1`.

Could you please verify this item? If there is a difference in scaling between the Open Market and Private Market cases we will have to investigate further.

Reply:
Hi @ix-certification I can verify what you describe is correct, using the current template for index, `pub_rev` only affects `ix_ox_pm `, so we are good.

What I was talking about before is - when I was trying to use the template described in README.md (the one you suggested me to undone, and I did undo it), I happened to find out the scale for `ix_ox_om` and `ix_ox_pm` is different,
![screen shot 2018-10-18 at 3 24 23 pm](https://user-images.githubusercontent.com/25375882/47187865-95ab1e80-d2ea-11e8-9762-259449d13463.png)
If you revert to commit 852285a and break at the same place. But I guess it's just how it is defined on your side, which is not related to the template OpenX is currently using.